news/2018-01-18-community-choice-2017.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Community Choice 2017"
+date: 2018-01-18 07:00:00 +0000
+---
+
+With 2017 well behind us, it's time to take a moment to sit back and reflect on which maps were truly the greatest of the year. Have your say and help us figure out which maps from 2017 are truly the Community's Favourite!
+
+[![](/wiki/shared/news/2018-01-18-community-choice-2017/banner.jpg)](https://osu.ppy.sh/community/contests/59)
+
+You may know them as the "best of" votes held historically throughout the past few years, but the votes are back for 2018 with a brand new look, and a better name!
+
+Each osu! player gets **10 votes per gamemode** to cast for their favourite ranked maps of 2017 that they've played. The voting is super simple - find your favourite 10 maps in any mode, shoot them a star, and you're done!
+
+The highest voted map for each gamemode will be declared the Community Favourite of 2017 and earn its mapper **a shiny new profile badge, and 6 months of osu!supporter** for their efforts.
```

Comment:
Game mode
Community's Favourite?

Reply:
Mapper into creator?

Reply:
community favourite is exalted singular so works proper in this instance